flavors/benchmark/benchmark.go

```diff
@@ -69,7 +71,9 @@ func NewBenchmark(cfg *config.Config) (Benchmark, error) {
 		}, nil
 	case config.CIS_GCP:
 		return &GCP{
-			IdentityProvider: &identity.Provider{},
+			IdentityProvider:     &identity.Provider{},
+			CfgProvider:          &auth.ConfigProvider{},
+			inventoryInitializer: &inventory.ProviderInitializer{},
```

Comment:
(not actionable in this PR observation) We don't really need `GCP` or any other `Benchmark` implementation to be exported. Until we actually need to do that, we can make everything private.